services/db.js

```diff
@@ -44,7 +44,13 @@ function get(key, testCacheEnabled) {
         // Parse non-uri data to match Postgres
         return isUri(key) ? data : JSON.parse(data);
       })
-      .catch(() => postgres.get(key));
+      .catch(() => {
+        return postgres.get(key)
+          .then(data => {
+            return redis.put(key, JSON.stringify(data))
+              .then(() => data);
+          });
```

Comment:
let's add a catch, so that if redis.put fails, we still return the data retrieved from postgres.put

Reply:
Good "catch".

https://github.com/clay/amphora-storage-postgres/pull/57/commits/45cabd3dcd3967fe0832b39d35ccd5f5a1421e6a

I added it in this commit ^ & verified the test by writing it to fail before I added the `.catch`.
I don't love that we are just swallowing the error message... but this library doesn't have any logging convention yet.

Should we throw in just a generic call to console.warn?
Integrate clay-logger?

Reply:
having some logging across this repo seems like a bigger task -- maybe a little console.warn for now would do the trick?

Reply:
I found where we have the `clay-log` util configured in the repository.
These cases are now logged.